src/windowsservercore/ltsc2019/webassembly/Dockerfile

```diff
@@ -34,3 +56,32 @@ RUN msiexec /i %TEMP%\nodejs.msi /quiet /passive /qn /norestart
 RUN npm install jsvu -g
 RUN npm exec -c "jsvu --os=win64 --engines=v8,spidermonkey"
 RUN setx PATH "%PATH%;%USERPROFILE%\.jsvu"
+
+# Install cmake
+ENV CMAKE_VERSION 3.20.3
+RUN curl -SL --output %TEMP%\cmake-win.zip https://github.com/Kitware/CMake/releases/download/v%CMAKE_VERSION%/cmake-%CMAKE_VERSION%-windows-x86_64.zip \
+    && mkdir C:\cmake \
+    && tar -C C:\cmake -zxf %TEMP%\cmake-win.zip \
+    && setx PATH "%PATH%;C:\cmake\cmake-%CMAKE_VERSION%-windows-x86_64\bin"
+
+# Download the Build Tools bootstrapper.
+ADD https://aka.ms/vs/16/release/vs_buildtools.exe vs_buildtools.exe
+
+# Visual Studio with .NET and C++ tools
+RUN mkdir C:\BuildTools
+
+RUN vs_buildtools.exe --quiet --wait --norestart --nocache \
+    --installPath C:\BuildTools \
+    --add Microsoft.VisualStudio.Workload.ManagedDesktopBuildTools \
+    --add Microsoft.VisualStudio.Workload.VCTools \
+    --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 \
+    --remove Microsoft.VisualStudio.Component.Windows10SDK.10240 \
+    --remove Microsoft.VisualStudio.Component.Windows10SDK.10586 \
+    --remove Microsoft.VisualStudio.Component.Windows10SDK.14393 \
+    --remove Microsoft.VisualStudio.Component.Windows81SDK \
+    --add Microsoft.VisualStudio.Component.Windows10SDK.17763 \
+|| IF "%ERRORLEVEL%"=="3010" EXIT 0
+
+# Define the entry point for the docker container.
+# This entry point starts the developer command prompt and launches the PowerShell shell.
+ENTRYPOINT ["C:\\BuildTools\\Common7\\Tools\\VsDevCmd.bat", "&&", "powershell.exe", "-NoLogo", "-ExecutionPolicy", "Bypass"]
```

Comment:
```suggestion
ENTRYPOINT ["%ProgramFiles(x86)%\\Microsoft Visual Studio\\2019\\BuildTools\\Common7\\Tools\\VsDevCmd.bat", "&&", "powershell.exe", "-NoLogo", "-ExecutionPolicy", "Bypass"]
```

Reply:
Updated this to not use an environment variable which isn't support for `ENTRYPOINT` instructions.